packages/server/src/application/setupTests.ts

```diff
@@ -5,3 +5,11 @@ import '../modules';
 
 process.env.TOKEN_SECRET = 'test_secret';
 beforeAll(() => application.init());
+
+beforeAll(async () => {
+  jest.spyOn(console, 'error').mockImplementation(jest.fn());
```

Comment:
You should do this in a specific test where you want to silence the console. This shouldn't be the default behavior.

Maybe it makes sense to extact mock in a specific function.

Reply:
I've put it in the used tests for now.

Maybe those tests testing authorisation (expecting 401) are not in the right place - removing them from there and make a separate test file testing only authorisation. Saying that because the silenced errors are for this error.

Reply:
That's ok, each route should verify with test it's secure.

Please also add a count of expected calls in the test.
```
// Expecting only single authorisation error
expect(console.error).toHaveBeenCalledTimes(x)
```

When you mock something so high level it can hide other errors/warning that you'd like to see in the test and potentially adjust logic.

Reply:
@MrBr done